i18n/es_ES.csv

```diff
@@ -36,10 +36,10 @@
 "Display only for selected IPs","Mostrar solo para IP seleccionadas"
 "Show method only to Magento set IPs.","Mostrar método de pago solo para IP establecidas por Magento."
 "Magento developer client restrictions.","Esta opción es para desarrolladores, y se usa cuando se quiere probar un método de pago específico sin mostrarlo públicamente."
-"Creditcards","Tarjetas de crédito"
-"Enable Creditcards","Activar tarjetas de crédito"
-"Allowed Creditcards","Tarjetas de crédito permitidas",
-"Select payment method specific Creditcards","Seleccionar tarjetas de crédito específicas del método de pago"
+"Credit and debit cards","Tarjetas de crédito"
```

Comment:
@Buckaroo-Rene the translation should be also updated, ex: "Tarjetas de crédito y débito" ?

Reply:
@Buckaroo-Rens